pulpcore/pulpcore/etc/pulp/server.yaml

```diff
@@ -85,6 +85,15 @@
 # server:
 #   working_directory: /var/cache/pulp
 
+# Content Application
+#
+# `content`: The content serving application.
+#   `web_server`: The type of web server.  Must be: (django|apache|nginx).
+#                 When set to 'apache', the X-SENDFILE header is injected which delegates
+#                 streaming the content to Apache.  Requires: mod_xsendfile to be installed.
+#                 when set to 'nginx', the X-Accel-Redirect header is injected which delegates
+#                 streaming the content to NGINX.
+
```

Comment:
For most settings we have the documentation of the settings and then a easy to uncomment actual settings sections. Like:
```
# `FOO`: this does somethings, can be set to foo or bar
# FOO: bar 
```